### ISSUE TITLE
Query and display file count on project contents page

### ISSUE THREAD

**Issue by bess** [pulibrary/tiger-data-app#391]
![Screenshot 2024-01-24 at 11 32 37 AM](https://github.com/pulibrary/tiger-data-app/assets/65608/d5b407b1-62ff-4cb7-a7f4-bec4951d803b)

- This is already calculated by Mediaflux, and stored within a metadata field for each Collection

Acceptance Criteria:
- [ ] Retrieve the count of files associated with the Collection from the Mediaflux server API
- [ ] Propagate this through the View

**Comment by hectorcorrea** [comment]
The aTerm command to retrieve the count of all files and directory in the collection and its subcollections is:

```
asset.count :where asset in static collection or subcollection of 1098
```

(where 1098 is the Mediaflux asset id)

**Comment by JaymeeH** [comment]
Hey @matthewjchandler , Do you want to have the total number of both files and collections under a project displayed? or just the total number of files?

**Comment by matthewjchandler** [comment]
@JaymeeH for this issue, I'm only concerned with the number of files.

My understanding of Mediaflux is collections within a project represent directories, which are addressed in #384.
